src/pages/home/sidebar/BottomTabBarFloatingActionButton/index.tsx

```diff
@@ -32,7 +32,6 @@ function BottomTabBarFloatingActionButton() {
     return (
         <FloatingActionButtonAndPopover
             ref={popoverModal}
-            // @ts-expect-error Error will be resolved after FloatingActionButtonAndPopover migration to Typescript
```

Comment:
But you migrated FloatingActionButtonAndPopover in this PR, right?

Reply:
its not there anymore so not sure why you saw it in diff 🤔